fedimint-server/src/db.rs

```diff
@@ -105,25 +89,7 @@ impl_db_record!(
 impl_db_lookup!(key = AlephUnitsKey, query_prefix = AlephUnitsPrefix);
 
 pub fn get_global_database_migrations() -> MigrationMap {
-    let mut mm = MigrationMap::new();
-    mm.insert(DatabaseVersion(0), |dbtx| migrate_to_v1(dbtx).boxed());
-    mm
-}
-
-/// Adds a database key that contains the current session count so that we don't
-/// have to scan the entire session history every time just to count the
-/// entries.
-async fn migrate_to_v1(dbtx: &mut DatabaseTransaction<'_>) -> Result<(), anyhow::Error> {
-    use futures::StreamExt;
-
-    let session_count = dbtx
-        .find_by_prefix(&SignedSessionOutcomePrefix)
-        .await
-        .count()
-        .await as u64;
-    dbtx.insert_new_entry(&SignedSessionOutcomeCountKey, &session_count)
-        .await;
-    Ok(())
+    MigrationMap::new()
```

Comment:
Why are we removing the migration? Wasn't this just recently added?

Reply:
Yes but the added db entry was redundant and we found a way to make it unnecessary.

Reply:
Since this wasn't part of any official release yet we can remove it.